.ci/podSpecs/distribution-template.yml

```diff
@@ -21,15 +21,13 @@ spec:
         - name: JAVA_TOOL_OPTIONS
           value: |
             -XX:+UseContainerSupport
-        - name: DOCKER_HOST
-          value: tcp://localhost:2375
       resources:
         limits:
-          cpu: 12
-          memory: 50Gi
+          cpu: 20
+          memory: 92Gi
```

Comment:
❓ Are these numbers based on something specific or just a gut feeling?
🙃 Would love to see a mention of these values in the commit message

Reply:
I downscaled the dind sibling and simply shifted the resources to the maven pod. The total number (i.e. the sum) is based on the node resources - we essentially grab the complete node by maxing out its resources. How it's divided was previously down empirically by testing to minimize build time and flakiness.

Reply:
I see, thanks for the explanation. 👍